Sources/Sham/Models/StubbedDataCollection.swift

```diff
@@ -0,0 +1,79 @@
+// Copyright © 2020 SpotHero, Inc. All rights reserved.
+
+import Foundation
+
+/// An object that can be injected into the launch environment to set up the mocking information.
```

Comment:
Can the language around the launch environment be removed here? `StubbedDataCollection` is just an object  for stubbing network requests which, by itself, doesn't have anything to do with the launch environment.